docs/source/index.mdx

```diff
@@ -21,24 +25,46 @@ description: A strongly-typed, caching GraphQL client for iOS, written in Swift
   </ButtonLink>
 </p>
 
-## Benefits
+## Key Features
```

Comment:
```suggestion
## Key features
```

Reply:
Pretty much all article titles and headings use sentence casing

Reply:
Oh really? That’s so odd to me… it feels so wrong to do that, just because I learned the other way and have been trained to do it since elementary school. 

But if that’s how we do it across the board, I’ll change my titles then!

Reply:
Haha yeah it's just the convention we happen to use! Nothing really right-er or wrong-er about it 😄